db/migrate/20161016195659_add_name_to_prices.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToPrices < ActiveRecord::Migration
```

Comment:
you already have name column when you create price table. I'll take of this @sbrks